pkg/manifests/manifests.go

```diff
@@ -612,6 +613,18 @@ func (f *Factory) PrometheusK8sServingCertsCABundle() (*v1.ConfigMap, error) {
 	return c, nil
 }
 
+func (f *Factory) PrometheusK8sCSRControllerCABundle(data map[string]string) (*v1.ConfigMap, error) {
```

Comment:
mini-nit: i think we are unit-testing all those factory methods. let's add a test for that one too.